src/main/java/com/gooddata/executeafm/resultspec/AttributeSortItem.java

```diff
@@ -20,21 +21,33 @@
  */
 @JsonTypeInfo(include = As.WRAPPER_OBJECT, use = Id.NAME)
 @JsonTypeName("attributeSortItem")
+@JsonInclude(JsonInclude.Include.NON_NULL)
 public class AttributeSortItem implements SortItem {
 
     private final String direction;
-
     private final String attributeIdentifier;
+    private final Aggregation aggregation;
 
     @JsonCreator
     public AttributeSortItem(@JsonProperty("direction") final String direction,
-                             @JsonProperty("attributeIdentifier") final String attributeIdentifier) {
+                             @JsonProperty("attributeIdentifier") final String attributeIdentifier,
+                             @JsonProperty("aggregation") final Aggregation aggregation) {
```

Comment:
We have the rule here "never use enum when deserializing, because they don't work with REST API changes (eg. new value added on the backend)" - see https://github.com/gooddata/gooddata-java/blob/master/CONTRIBUTING.md ... you can still use it in java API (constructors), but convert field to String plz.